RELEASE_NOTES.rst

```diff
@@ -1,3 +1,12 @@
+Unreleased changes
+-----
+
+Bug fixes:
+
+* https://github.com/gymreklab/TRTools/issues/146 fixed record positions being compared twice
+* HipSTR records are now compared by starting position of the repeat, not the whole called allele
```

Comment:
Can we clarify - in which tools, and what about flanking bps on the end?

Reply:
I've updated the file to better describe my changes. 
Starting position of the record is only used to determine whether the two records are comparable. The comparison itself works the same as before (it already uses harmonized data)